### ISSUE TITLE
No return type for Controller::get

### ISSUE THREAD

**Issue by binarious** [phpstan/phpstan-symfony#8]
Same error as described here: https://github.com/lookyman/phpstan-symfony/issues/9

Code:
```php
$this->get('session')->getFlashBag()->set('notice', 'Message');
```

Error:
```
Call to an undefined method object::getFlashBag().
```

It does work with `$this->container->get('session')`.

- phpstan/phpstan-symfony: 0.10.1
- phpstan: 0.10.2

**Comment by binarious** [comment]
Nevermind. We're extending `AbstractController` which doesn't support this (while still working):

> The difference is that when you extend AbstractController, you can't access services directly via $this->get() or $this->container->get().

**Comment by TomasVotruba** [comment]
I think I had similar bug in Rector, where PHPStan Analyzer is used.
My case whas, that the `AbstractController` uses `ControllerTrait` and the trait has `get()` method. PHPStan was unable to load parent class trait's method, so I had create own wrapper in reflection for doing this.

**Comment by lookyman** [comment]
Fixed in 0.10.2
